client/messaging/common-message-layout.tsx

```diff
@@ -29,10 +29,11 @@ const newDayFormat = new Intl.DateTimeFormat(navigator.language, {
   day: '2-digit',
 })
 
-const Username = styled.span`
+const Username = styled(ConnectedUsername)`
   ${titleSmall};
 
   margin-right: 8px;
+  padding: 4px 0;
```

Comment:
nit: use `padding-block` for this

I feel like `padding-block`/`padding-inline` are a better choice for stuff like this because it doesn't override the value we don't care about (so it's more resilient to changes in the thing we're modifying)